content/en/core/releases/4.4.0.md

```diff
@@ -0,0 +1,103 @@
+---
+
+## Known issues
+
+Check the repository for the [latest known issues](https://github.com/medic/cht-core/issues?q=is%3Aissue+label%3A%22Affects%3A+4.4.0%22).
+
+## Upgrade notes
+
+### Breaking changes
+
+None.
+
+### UI/UX changes
+
+#### CHT dialogs.
+
+CHT dialogs were updated to align with Material guidelines. This includes updating the visual design / placement of dialogs as well as updating the default text of dialogs as necessary.
+
+{{< figure src="../images/4.4.0-8391-cht-dialogs.png" class="left col-12 col-lg-12" >}}
+<br clear="all">
+
+- [#8390](https://github.com/medic/cht-core/issues/8390): Align "dialog" components with Material design guidelines
+
+
+## Highlights
+
+### Modals\'s text
+
+Some modals\'s text have been improved to increase understanding and some keys have been replaced.
+
+- Bulk delete reports
+  - `bulkdelete.confirm.title` used for the modal title when deleting 1 report
+  - [NEW] `bulkdelete.confirm.title.plural` used for the modal title when deleting multiple reports.
+- Edit message group
+  - [NEW] `edit_message_group.modal.title`: used for the modal title
+- Report a bug
+  - [NEW] `feedback.modal.submit`: used for the modal submit button label. 
+- Confirm Navigation - like when exiting a form that hasn't been submitted
+  - [NEW] `confirm.destructive.navigation.title`: used for the modal title
+  - [NEW] `confirm.destructive.navigation.submit`: used for the modal submit button label.
+- Training cards
+  - [NEW] `training_cards.confirm.title`: used for the modal title
+- Confirm verifying a report
+  - [NEW] `confirm.verification.title`: used for the modal title
+  - [NEW] `confirm.verification.submit`: used for the modal submit button label.
+
+We have provided translations for all of these in the CHT Core supported languages (English, Spanish, French, Indonesian, Hindi, Nepali, Bambara, Swahili)
+
+### Features
+
+None.
+
+### Improvements
+
+- [#6514](https://github.com/medic/cht-core/issues/6514): Migrate from SW-Precache to Workbox
+- [#8179](https://github.com/medic/cht-core/issues/8179): nginx and haproxy should respect `Accept` headers when responding with error templates
+- [#8390](https://github.com/medic/cht-core/issues/8390): Align "dialog" components with Material design guidelines
+- [#8442](https://github.com/medic/cht-core/issues/8442): Telemetry geolocation:failure events for geolocation should be unique
+- [#8498](https://github.com/medic/cht-core/issues/8498): Remove targets disable admin option
+- [#8506](https://github.com/medic/cht-core/issues/8506): Update browser support notice to match official versions
+- [#8511](https://github.com/medic/cht-core/issues/8511): Update configuration deployment action to use latest cht-conf
+
+### Security fixes
+
+- [#8443](https://github.com/medic/cht-core/issues/8443): Security issue 8156
+
+### Performance improvements
+
+- [#8238](https://github.com/medic/cht-core/issues/8238): Figure out a different way to use purged docs cache
```

Comment:
Here this can be better explained as: `store purged docs cache in couchdb `
I think this is something that app builders want to know a bit more about, from the PR: `each user gets their own doc in a new database medic-cache to store purged document ids the database is destroyed (deleted and recreated) every time purging runs`

Reply:
Thank you for the details! ✨